dashboard/app/helpers/users_helper.rb

```diff
@@ -116,4 +116,15 @@ def level_info(user, script_level, user_levels)
       0
     end
   end
+
+  def options_for_user_enrollment_users
+    options = []
+
+    User.all.each do |course|
```

Comment:
Actually - I'll just make the user type in their email for now and use that. Easier.